src/ui/oled/oled.c

```diff
@@ -134,27 +135,30 @@ void oled_init(void)
 
     // Initialize
     _write_command(OLED_CMD_SET_DISPLAY_OFF);
-    _write_command(OLED_CMD_SET_LOW_COL(0));
-    _write_command(OLED_CMD_SET_HIGH_COL(0));
-    _write_command(OLED_CMD_SET_PAGE_START_ADDRESS(0));
-    _write_command(0xdc); /*set display start line*/
-    _write_command(0x00); // +0x7f
     _write_command(OLED_CMD_SET_CONTRAST_CONTROL_FOR_BANK0);
-    _write_command(0x80); /* 0x00..0xff */
```

Comment:
could you please split this particular change into a new commit and combine it with using `OLED_CMD_SET_CONTRAST_CONTROL_FOR_BANK0` in `oled_brightness()`, and using `oled_brightness()` here? for clarity.

Reply:
I removed that function instead. It was dead code and I'll have to make larger changes to this code anyway due to the bitboxbase.